docs/user/openstack/external_load_balancer.md

```diff
@@ -0,0 +1,224 @@
+        vips:
+          - 192.168.10.7
+        min_backends: 1
+        healthcheck: "httpchk GET /healthz/ready HTTP/1.0"
+        frontend_port: 80
+        haproxy_monitor_port: 8082
+        balance: roundrobin
+        backend_opts: "check check-ssl port 1936 inter 1s fall 2 rise 3 verify none"
+        backend_port: 80
+        backend_hosts: *lb1_hosts
+      - name: ingress_https
+        vips:
+          - 192.168.10.7
+        min_backends: 1
+        healthcheck: "httpchk GET /healthz/ready HTTP/1.0"
+        frontend_port: 443
+        haproxy_monitor_port: 8083
+        balance: roundrobin
+        backend_opts: "check check-ssl port 1936 inter 1s fall 2 rise 3 verify none"
+        backend_port: 443
+        backend_hosts: *lb1_hosts
+      - name: mcs
+        vips:
+          - 192.168.10.5
+        min_backends: 1
+        frontend_port: 22623
+        haproxy_monitor_port: 8084
+        balance: roundrobin
+        backend_opts: "check check-ssl inter 5s fall 2 rise 3 verify none"
+        backend_port: 22623
+        backend_hosts: *lb1_hosts
+```
+
+**NOTE:** Note that this configuration doesn't cover the BGP capabilities of the Ansible role and it's suggested to read the role manual to learn more about it.
+
+6. Run Ansible to deploy the load balancer:
+
+```bash
+ansible-playbook -i inventory.yaml -e "@vars.yaml" playbook.yaml
+```
+
+7. To verify that your load balancer has been deployed, you can SSH into it and open `/etc/haproxy/haproxy.cfg`. It should be populated for the different backends.
+
+
+## Deploy OpenShift
+
+Now that your load-balancer(s) are ready, you can deploy OpenShift.
+Here is an example of an `install-config.yaml`:
+
+```yaml
+apiVersion: v1
+baseDomain: mydomain.test 
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: m1.xlarge
+  replicas: 3
+metadata:
+  name: mycluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.10.0/24
+platform:
+  openstack:
+    cloud: mycloud
+    machinesSubnet: 8586bf1a-cc3c-4d40-bdf6-c243decc603a
+    apiVIPs:
+    - 192.168.10.5
+    ingressVIPs:
+    - 192.168.10.7
+    loadBalancer:
+      type: UserManaged
+featureSet: TechPreviewNoUpgrade
+```
+
+* `loadBalancer` is a new stanza created in OCP 4.13. The default type is `OpenShiftManagedDefault` (which will deploy HAproxy and Keepalived in OCP, known as the internal load balancer). Setting it to `UserManaged` will allow an external load balancer to replace the internal one.
+* `machinesSubnet` is the subnet ID where both the OpenShift cluster and the external load balancer are deployed.
+* `TechPreviewNoUpgrade` featureSet has to be enabled, since the feature is currently on TechPreview. It'll be GA at some point.
+
+
+Deploy the cluster:
+```bash
+openshift-install create cluster --dir ocp
+```
+
+## Known limitations
+
```

Comment:
Rather than stating that each of the items below is in our road map, maybe an over arching statement that states that each limitation is currently being addressed.